python/tvm/hybrid/utils.py

```diff
@@ -0,0 +1,109 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Helper functions in Hybrid Script Parser"""
+
+import inspect
+from tvm import IRModule
+
+from . import _ffi_api
+from .parser import source_to_op
+
+
+def create_module(functions=None):
```

Comment:
Why `functions` are optional? Is that mean we will create an empty module if `functions=None`

Reply:
Yes.